### ISSUE TITLE
nfs-resizer container crashes with CSI driver neither supports controller resize nor node resize 

### ISSUE THREAD

**Issue by JPRbrs** [kubernetes-csi/external-resizer#179]
Hello, 

I am using nfs-resizer as part of  manila-csi (1.22.0) and csi-nfs-driver (mcr.microsoft.com/k8s/csi/nfs-csi:latest) configuration.
manila-csi-openstack-manila-csi-controllerplugin-0  pod is crashing because of the following error being thrown by nfs-resizer:

`I1116 14:06:34.639310       1 main.go:90] Version : v1.2.0                                                              
I1116 14:06:34.645487       1 common.go:111] Probing CSI driver for readiness                                           
F1116 14:06:34.656982       1 main.go:158] CSI driver neither supports controller resize nor node resize                
goroutine 1 [running]:                                                                                                  
k8s.io/klog/v2.stacks(0xc00013a001, 0xc00053e000, 0x69, 0xa0)                                                           
        /workspace/vendor/k8s.io/klog/v2/klog.go:1021 +0xb9                                                             
k8s.io/klog/v2.(*loggingT).output(0x27fdaa0, 0xc000000003, 0x0, 0x0, 0xc0005b4000, 0x208b4d3, 0x7, 0x9e, 0x40e000)      
        /workspace/vendor/k8s.io/klog/v2/klog.go:970 +0x191                                                             
k8s.io/klog/v2.(*loggingT).printDepth(0x27fdaa0, 0xc000000003, 0x0, 0x0, 0x0, 0x0, 0x1, 0xc0000275f0, 0x1, 0x1)         
        /workspace/vendor/k8s.io/klog/v2/klog.go:733 +0x16f                                                             
k8s.io/klog/v2.(*loggingT).print(...)                                                                                   
        /workspace/vendor/k8s.io/klog/v2/klog.go:715                                                                    
k8s.io/klog/v2.Fatal(...)                                                                                               
        /workspace/vendor/k8s.io/klog/v2/klog.go:1489                                                                   
main.main()                                                                                                             
        /workspace/cmd/csi-resizer/main.go:158 +0x123d                                                                  
`

As far as I can see in csi-nfs-driver does not support resizing but shouldn't in this case nfs-resizer just somehow ignore the resize requests? Or at least not fail until a resize request is issued?

In summary, I would like to know if anyone was able to make resize work (I've seen some comments that indicate that it is possible) If it is not possible, how can I make nfs-resizer not crash?

Many thanks in advance

**Comment by JPRbrs** [comment]
It looks like I wasn't really using manila-csi-plugin version 1.22. Bumping it to that version prevented nfs-resizing from crashing.

For the curious, I think it was due to the way I'm deploying the chart along csi-nfs-driver. They are both deployed as part of the same parent chart, and the fact that the tag for manila-csi-plugin tag is taken from the .Chart.AppVersion messes up with it
